ui/lib/core/addon/components/replication-page.js

```diff
@@ -29,22 +30,45 @@ export default Component.extend({
     }
     this.set('reindexingDetails', resp);
   }),
+  isSummaryDashboard: computed('model.dr.{mode}', 'model.performance.{mode}', function() {
+    const router = this.router;
+    const currentRoute = router.get('currentRouteName');
+
+    // we only show the summary dashboard in the replication index route
+    if (currentRoute === 'vault.cluster.replication.index') {
+      const drMode = this.model.dr.mode;
+      const performanceMode = this.model.performance.mode;
+      return drMode && performanceMode === 'primary';
+    }
+  }),
   formattedReplicationMode: computed('model.{replicationMode}', function() {
```

Comment:
Do you need to also compute based off `isSummaryDashboard`? Can you compute from a computed property? 🤔